### PR TITLE
Web hosting - Case Study

### PR DESCRIPTION
**Pull request by kardokiyani**
New PR to get a request review from my coach.

### CODE REVIEW

File: web-hosting-case-study.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+
+- One of the strengths with VPS is that it works better than shared hosting because it has improved performance and security, something shared hosting don`t have.
+
+- It`s cost benefit, it saves you money.
+
+- Get the acquisition and maintenance of the equipment, and you will get reduction in the cost.
+
+### Dedicated Server Hosting
+
+- The biggest strength is that it`s very safe, zero risk of being harmed by other companies.
```
Comment (Fermain) [inline web-hosting-case-study.md:224]:
```suggestion
- The biggest strength is that it's very safe, zero risk of being harmed by other companies.
```

This is quite a bold statement... Do you have a source for it?

**Review by Fermain — CHANGES_REQUESTED** [review]
This is a detailed case study with enough information present. The vast majority of my comments are grammar or punctuation changes - but I have not been able to find all of these.

I suggest using this workflow to help (I use this for writing too):

- Install code spell checker in vs code and set this to English
- Install grammarly for vs code and set this to English
- Ask ChatGPT act as your copyeditor:

"Hi GPT, please behave as my subeditor and correct any punctuation or grammar mistakes in this text without changing the meaning."

This will eradicate the majority of the remaining issues and make it easier to focus on the content. I would like to review this again once these issues are resolved.

File: web-hosting-case-study.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+
+### 2008 -  GitHub Pages is launched.
+
+Github created Github Pages in 2008, it`s a static web hosting service for project documentation, blogs and books etc. Github pages content is stored in a Git repository in two kinds of way, you can do it as files served to visitors verbatim, and the other way is in a markdown format.Github works trough Jekyll static website an blog generator. Basically each time the content source is updated, Jekyll regenerates the website  and automatically  serves it via Github Pages tool. Github have both free and  and paid tiers of service, which makes it very popular for school and work related stuff. Github pages supports HTTPS encryption and one one more thing to notice is that websites generated through this service are either hosted as subdomains of the Github.io domain or it can be connected to custom  domains which is bought through a third-party domain name registrar.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+
+#### A2 Hosting -High-powered web hosting with plans to suit your unique needs
+
+## Features
+
+![features-image](https://www.upvoty.com/wp-content/uploads/2020/09/features.png)
+
+### Disk Space
+
+Disk space is basically the total amount of data that a hard disk/hard drive can store. Disk capacity is displayed usually in three different ways:
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md:108]: I have answered the answer to this question in the file. Fixed it.

File: web-hosting-case-study.md [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+
+- MB (megabytes)
+
+- GB (gigabytes)
+
+- TB (terabytes)
+
+### Bandwidth
+
+With Bandwidth feature, you can load your website faster, because loading time can take tme and is the time it takes to transfer different website files.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: I have answered the answer to this question in the file. Fixed it.

File: web-hosting-case-study.md [hunk 5]
```diff
@@ -0,0 +1,372 @@
...
+
+- Secure shell (SSH) access
+
+- Multi-factor authentication
+
+- Domain privacy
+
+### Uptime
+
+What Uptime do is that it measures te percentage of time your website is up and available to the users who visit your site. With other words, the closer to 100% uptime, the better. If you use the best web hosting, it van provide high uptimes by offering backup servers and backup servers in multiple locations. Many web hosts have uptime, one example is Bluehost, which have 99.9% uptime.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 6]
```diff
@@ -0,0 +1,372 @@
...
+
+### Website Speed
+
+Website speed helps loading your website faster, if you are using the right tools. Not only does website speed affect user experience, but it also does affect search engine optimization (SEO). With good web hosting companies, you can use:
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed the grammar. And answered the answer to this question in the file.

File: web-hosting-case-study.md [hunk 7]
```diff
@@ -0,0 +1,372 @@
...
+
+- Using servers with solid-state drives (SSD)
+
+- Limiting sites on a shared server
+
+- Offering a content delivery network (CDN) service
+
+With these right tools above, you can help users deliver quick page.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 8]
```diff
@@ -0,0 +1,372 @@
...
+
+- One-Click Installer
+
+- File Transfer Protocol
+
+- E-Commerce Features
+
+### Domain Names
+
+Your domain name is the digital address of your digital space on the internet. The reason behind making this was to find and identify internet entities, but without using IP addresses. A domain name is basically an Internet resource name that is universally understood by Web servers and online organizations and provides all pertinent destination information. This makes everything more safe and organized.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md:178]: I have answered the answer to this question in the file. Fixed it.

File: web-hosting-case-study.md [hunk 9]
```diff
@@ -0,0 +1,372 @@
...
+
+### Website Builder
+
+Website builder is a tool or program that can help you create a website without actual manual code editing. There is two times of website builders:
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 10]
```diff
@@ -0,0 +1,372 @@
...
+
+- Chatbots
+
+- Email support
+
+- Calls
+
+### Backups 
+
+Backups is very important. It can prevent data loss, which is very important. It does not matter how good your website is, a website can always get hit by something which will take the website down. A good web hosting company should restore your website with minimal downtime and make frequent automatic backups of your website to like I said prevent data loss.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 11]
```diff
@@ -0,0 +1,372 @@
...
+
+## Strengths
+
+![strengths-image](http://nejcslovnik.com/wp-content/uploads/2016/09/Image_web.jpg)
+
+### Shared Hosting
+
+- Low cost: one of the strengths with web hosting is low cost, , thats one of the main advantages.
+
+- Easy to use: it`s easy to use web hosting where it is simple to install and maintain the site trough a system where you just "click and drag" when facilitating the process.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 12]
```diff
@@ -0,0 +1,372 @@
...
+
+- Technical Support: the good thing is that the technical support is guaranteed by most of the  companies that offer web hosting, so it`s not scary or a risk to do it if you are not sure. Here the user will get help with important factors, such as the website itself, the server and configuration requirements.
+
+- Resources: another advance is resources, here you can get a collection of resources that can grow and get bigger while following the websites requirements and demands.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 13]
```diff
@@ -0,0 +1,372 @@
...
+
+### VPS Hosting (Virtual Private Server)
+
+- One of the strengths with VPS is that it works better than shared hosting because it has improved performance and security, something shared hosting don`t have.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it. I have answered the answer to this question in the file.

File: web-hosting-case-study.md [hunk 14]
```diff
@@ -0,0 +1,372 @@
...
+
+- It`s cost benefit, it saves you money.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 15]
```diff
@@ -0,0 +1,372 @@
...
+
+- Get the acquisition and maintenance of the equipment, and you will get reduction in the cost.
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md]: Fixed it.

File: web-hosting-case-study.md [hunk 16]
```diff
@@ -0,0 +1,372 @@
...
+
+Your domain name is the digital address of your digital space on the internet. The reason behind making this was to find and identify internet entities, but without using IP addresses. A domain name is basically an Internet resource name that is universally understood by Web servers and online organizations and provides all pertinent destination information. This makes everything more safe and organized.
+
+### Website Builder
+
+Website builder is a tool or program that can help you create a website without actual manual code editing. There is two times of website builders:
+
+- Offline Website Builders
+
+- Online Website Builders
```
[the remark this answers is not in the document]
Comment (kardokiyani) [reply web-hosting-case-study.md:188]: I have answered the answer to this question in the file. Fixed it.

File: web-hosting-case-study.md (hunk 1 above)
Comment (kardokiyani) [reply web-hosting-case-study.md:260]: Fixed the grammar. And answered the answer to this question in the file.

**Review by Fermain — APPROVED** [review]
A highly detailed case study on hosting. It would have been nice if you teamed up with Gonzalo who also covered this topic. Feedback provided during review was handled well, showing the ability to reflect and respond to criticism. Thanks for your contribution!

**Pull request merged**